components/sup/src/manager/file_watcher.rs

```diff
@@ -1293,17 +1293,29 @@ impl<C: Callbacks, W: Watcher> FileWatcher<C, W> {
         }
     }
 
-    pub fn run(&mut self) -> Result<()> {
+    /// Runs the FileWatcher in a loop.
+    ///
+    /// The `initial_file_event` parameter controls whether an event is emitted when the file was
+    /// already there at the time the FileWatcher was started.
+    pub fn run(&mut self, initial_file_event: bool) -> Result<()> {
```

Comment:
If we decide to go that way, then I'd prefer the `initial_file_event` parameter to have a default value set to `true`. That way, the callers do not need to be changed.

Reply:
That would be great. But how do you do that in rust?